packages/validator/src/validators/regexpValidator.ts

```diff
@@ -2,19 +2,19 @@ import * as logger from '../utils/logger';
 import { Log } from '../types';
 
 /**
- * Checks if key or value matches regular expression in specsStruct
+ * Checks if key or value matches regular expression in template
  * @param specs - specification that is being validated
- * @param specsStruct - validator specification structure containing the regular expression
+ * @param template - template containing the regular expression
  * @param paramPath - string of parameters separated by ".", representing path to current specs location
  * @param isKeyRegexp - determines if key or value should be checked
  * @returns incorrect formatting message if the parameter key or value did not match regular expression
  */
-export function validateRegexp(specs: any, specsStruct: any, paramPath: string, isKeyRegexp = false): Log[] {
+export function validateRegexp(specs: any, template: any, paramPath: string, isKeyRegexp = false): Log[] {
   const messages: Log[] = [];
 
   if (isKeyRegexp) {
```

Comment:
It seems like you're missing the case for when `isKeyRegexp` is false?

Reply:
Actually, on second look, it appears that when `isKeyRegexp` is false, none of the if-statements are executed, only the final `return messages;` which should be empty, correct? 

If so, could you make the code more explicit in that regard? E.g. just changing the structure to if-else I think would make the code easier to follow.

Reply:
Yes you are right, I use this style based on [Andre's recommendation](https://github.com/api3dao/airnode/pull/69#discussion_r495812979), so this is on many places, im not sure if I want to refactor this, but if you would think on some places it causes serious problems, let me know.

Reply:
Haha, yea I see me and Andre have opposite opinions on the issue. Yea, I'd leave it as-is.